frontend/packages/core/src/Input/stories/toggle-button-group.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import type { Meta } from "@storybook/react";
+
+import ToggleButtonGroup from "../toggle-button-group";
+
+export default {
+  title: "Core/Input/ToggleButtonGroup",
+  component: ToggleButtonGroup,
+} as Meta;
+
+export const ExclusiveDemo = () => {
+  const [value, setValue] = React.useState("MEOW");
+  const onChange = (_event: React.ChangeEvent<{}>, newValue: string) => {
+    // Note that we check for null because we want to enforce that
+    // at least value is active
+    if (newValue !== null) {
+      setValue(newValue);
+    }
+  };
```

Comment:
This makes me think that in the component, if the value is specified as an invalid option, we should handle null values such that either we leave the existing value(s) or if there is no value and exclusive is true we should default to the first option

Reply:
Are we saying that we should iterate over the children checking if there is a possible `value` prop, put those into an object, then check if the key exists? I guess I'm confused because the component doesn't seem to know about its chlidren

Reply:
Oh I think I get it. You mean like the `onChangeMiddleware` func

Reply:
Here I'm trying to say that we shouldn't need to do a null value check in the parent components that control the value being sent to `ToggleButtonGroup`. In the `ToggleButtonGroup`, if the `value` prop is null the component should either:
 * if the `multiple` prop is `true`, allow a value of null for the `value` prop
 * if the `multiple` prop is `false`, default to the first child since at least one value must be selected.

Reply:
hm for some reason children[0] is always undefined even though children seems to have 3 entries in it. Need to investigate more. Can't print the object though because apparently its a circular reference.

Reply:
thought more about this and it seems the proposed behavior is unintuitive. If a user clicks a choice to deselect it but they can't (because its not multiple), then the first value will become selected. It seems the more intuitive behavior would be that they just wouldn't be able to deselect any (swallow nulls if exclusive is true (multiple is false))

Reply:
it works for the exclusive case if I do `value={value || (multiple ? value : React.Children.toArray(children)[0].props.value)}` . However, this breaks when `multiple` is true because internally the way the component works is that its splicing together the selection choices.